x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/index.tsx

```diff
@@ -6,6 +6,7 @@
  */
 
 import React from 'react';
+import { isEmpty } from 'lodash';
```

Comment:
consider changing this to

```
import { isEmpty } from 'lodash/fp';
```

for consistency with other imports of `isEmpty`

Reply:
Kibana wants us to use lodash directly

> @XavierM I talked to @ppisljar and we're not sure about the rollout plans of EQL outside of security solution. We want to talk to @stacey-gammon about this as well.
> If you want this change to make 7.12, I would suggest that you copy the component over to sec. solution.
> Let me know what you think.

it won't work because I need to pick EQL from the language switcher in the query bar